frappe/commands/utils.py

```diff
@@ -489,12 +489,20 @@ def jupyter(context):
 @pass_context
 def console(context):
 	"Start ipython console for a site"
+	import warnings
+
 	site = get_site(context)
 	frappe.init(site=site)
 	frappe.connect()
 	frappe.local.lang = frappe.db.get_default("lang")
 
 	import IPython
+	from IPython.terminal import embed
+	from traitlets.config import get_config
+	terminal = embed.InteractiveShellEmbed()
+	terminal.extension_manager.load_extension("autoreload")
+	terminal.run_line_magic("autoreload", "2")
```

Comment:
I seem to recall that this autoreload extension doesn't play nicely with Frappe in certain conditions. Why make this default? If you must have it then it can also reside in your ipy startup config.

Reply:
Can you elaborate on the conditions please? I was thinking the opposite:

> Why _not_ make this default?

Reply:
The risk of a random crash is kinda big deal if you're doing something on a production site. 

Maybe make this configurable? (or reuse developer_mode as the configuration key)

Reply:
I've updated the PR Description with caveats.

Reply:
If it is turns out to be unstable, we can move it to a separate config.

Reply:
Why not `bench console --auto-reload` or something similar instead of dealing with configs and other magic?